src/pages/test-report/cat-home-test/components/vehicle-checks/vehicle-checks.html

```diff
@@ -0,0 +1,7 @@
+<div class="vehicle-checks-competency">
```

Comment:
we shouldn't need to duplicate vehicle-checks, there's nothing unique in the home tests, we should be using a common one made from b (Make B common and reuse it)

Reply:
Just looked at the designs and it's the same vehicle checks that we use for B+E and C so we should be using the common one rather then the duplication

Reply:
as per our discussion, this will be addressed at a later stage due to the lack of a vehicle checks common component, this task already exists in the technical debt document so it has not been edited as a result of this